app/controllers/application_controller.rb

```diff
@@ -5,6 +5,8 @@ class ApplicationController < ActionController::Base
   include ValidRequest
   include Pundit
 
+  rescue_from ActionView::MissingTemplate, with: :not_found
```

Comment:
Hmm should we only do this in production?

Reply:
Interesting, I still get the `ActionView::MissingTemplate` error when I load formats that have no templates in development.

Reply:
Oh interesting, I actually did not test it in development haha. Maybe it's not worth adding an `if Rails.env.production?` statement then.

Reply:
I find `ActiveRecord::NotFound` misleading here because it seems not to be related to ActiveRecord. Shouldn't smth like `ActionController::RoutingError ` be more suitable (+ adding it to exceptions)?

Reply:
I'm game for that! And HB already automatically ignores that error as well